replit_river/session.py

```diff
@@ -353,13 +357,14 @@ async def send_message(
                     await self.close(True)
                     return
                 async with self._ws_lock:
-                    if await self._ws_wrapper.is_open():
-                        # if it is not open it's fine, we already put it the buffer
-                        await self._send_transport_message(
-                            msg,
-                            self._ws_wrapper.ws,
-                            prefix_bytes=self._transport_options.get_prefix_bytes(),
-                        )
+                    if not await self._ws_wrapper.is_open():
+                        # If the websocket is closed, we should not send the message
+                        return
```

Comment:
The Websocket API throws an exception if you `send()` on a closed socket. Is there a reason we want to silently drop the message instead? Just an abstract concern, I don't have enough context to say it's an actual problem.

Reply:
we buffer anyways so its ok to not send altogether